components/telemetry-operator/controller/tracepipeline/resources.go

```diff
@@ -177,6 +177,7 @@ func makeProcessorsConfig() map[string]any {
 			"timeout":             "10s",
 			"send_batch_max_size": 512,
 		},
+		"filter": makeFilterProcessorConfig(),
```

Comment:
Shouldn't there be a unit test assuring that the filter is being generated?

Reply:
missing unit test added